server/migrations/20190502122038_create_authors.js

```diff
@@ -0,0 +1,12 @@
+
+
+exports.up = (knex, Promise) => Promise.all([
```

Comment:
Write the migrations following the same format as the first one.(Don't use Promise.all, let's be consistent) Also, `create_authors_table` should be the file name.